projects/step-frontend/src/lib/modules/timeseries/discover/attribute-stats/attribute-stats.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { AttributeStats, TimeSeriesService } from '@exense/step-core';
+
+@Component({
+  selector: 'step-discover-attribute-stats',
+  templateUrl: './attribute-stats.component.html',
+  styleUrls: ['./attribute-stats.component.scss'],
+})
+export class DiscoverAttributeStatsComponent implements OnInit {
+  isLoading = true;
+
+  @Input() attributeValues!: AttributeStats[];
```

Comment:
No action required, but please note, that **after** updating to A16 we should mark this as required:

@Input({ required: true })

Reply:
@neogucky Property 'attributeValues' has no initializer and is not definitely assigned in the constructor.. Do I miss something?